src/timessquare/domain/page.py

```diff
@@ -74,7 +75,10 @@ def read_ipynb(source: str) -> nbformat.NotebookNode:
         The notebook is read according to the `NB_VERSION` notebook version
         constant.
         """
-        return nbformat.reads(source, as_version=NB_VERSION)
+        try:
+            return nbformat.reads(source, as_version=NB_VERSION)
+        except Exception:
+            raise PageNotebookFormatError
```

Comment:
I think you may be losing the specific error message from `nbformat` saying what's invalid about the notebook, which might be frustrating for debugging. Usually what I do in cases like this (with vo-cutouts, for instance) is extract the name of the exception type and its string value and pass those in as parameters to the richer exception I'm raising, and use that to give it a more informative string value.